### PR TITLE
Create an IndividualElement class

### PR DESCRIPTION
**Pull request by goneall**
This will allow individuals to be of a type that can be instantiated and could resolve the issue with SHACL validation failing https://github.com/spdx/spec-parser/issues/165

### CODE REVIEW

**Comment by goneall** [comment]
See this conversation in https://github.com/spdx/spdx-3-model/issues/936 for context

**Comment by JPEWdev** [comment]
This might make it kinda hard to know how named individuals are supposed to be used; I think maybe we can deal with it if necessary but I need to do some testing before we go ahead and make this change

**Comment by JPEWdev** [comment]
Ah, you know what, I was think of enums. This should be fine

File: model/Core/Classes/IndividualElement.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+## Summary
+
+A concrete subclass of Element used by Individuals in the
+Core profile.
+
+## Description
+
+Individuals, such as NoneElement and NoAssertionElement, need to reference a
+concrete subclass of ELement.
```
Comment (bact) [inline model/Core/Classes/IndividualElement.md:13]:
```suggestion
concrete subclass of Element.
```

**Review by bact — CHANGES_REQUESTED** [review]

File: model/Core/Classes/IndividualElement.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+This class provides the type used by the individuals.
```
Comment (bact) [inline model/Core/Classes/IndividualElement.md:15]:
```suggestion
This class provides the type used by the individuals.

For individuals that are subclass of AnyLicenseInfo, please use IndividualLicensingInfo.
```

As AnyLicenseInfo is also a subclass of Element, do we have to point that out here as well?

**Review by bact — CHANGES_REQUESTED** [review]

File: model/Core/Classes/IndividualElement.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+A concrete subclass of Element used by Individuals in the
+Core profile.
+
+## Description
+
+Individuals, such as NoneElement and NoAssertionElement, need to reference a
+concrete subclass of Element.
+
+This class provides the type used by the individuals.
```
Comment (zvr) [inline model/Core/Classes/IndividualElement.md:15]:
```suggestion
This class provides the type used by the individuals defined in the Core profile.
```

**Review by zvr — CHANGES_REQUESTED** [review]

File: model/Core/Classes/IndividualElement.md (hunk 2 above)
Comment (zvr) [reply model/Core/Classes/IndividualElement.md:15]: We shouldn't be talking about individuals defined in other profiles here.

File: model/Core/Classes/IndividualElement.md (hunk 3 above)
Comment (bact) [reply model/Core/Classes/IndividualElement.md:15]: +1 to this for scoping.

**Review by bact — APPROVED** [review]

**Comment by bact** [comment]
Approved, providing the suggested "for Core profile" by @zvr is added to the Description.

**Comment by goneall** [comment]
@zvr - Accepted suggestion - if you could approve and merge everything should be working.

**Review by zvr — APPROVED** [review]

**Pull request merged**